hydra-web/src/js/modules/alias.js

```diff
@@ -131,7 +131,7 @@ function(
                     "bVisible":true,
                     "bSearchable":true,
                     "mRender":function(val,type,data){
-                        return val.join(",");
+                        return "<a href='#jobs/"+encodeURIComponent(val)+"/conf'>"+val+"</a>";
```

Comment:
Hydra allows multiple job ids to be mapped to the same alias. It was for an old use case. Although it's not used anymore, the right thing to do is to generate a separate job link for each element in the array.